classes/DatabaseTableModel.php

```diff
@@ -383,6 +384,11 @@ protected function loadColumnsFromTableInfo()
                 'id' => $columnName,
             ];
 
+            // Format quoted "null" values with quotes
+            if ($column->getNotnull() === false && strtolower($item['default'] === 'null')) {
```

Comment:
@bennothommo maybe you've misplaced the brackets: `strtolower($item['default']) === 'null'`?

Reply:
@w20k Oops :P